### DISCUSSION TITLE
"Create list" is greyed out in GitHub to add new starred lists

### DISCUSSION THREAD

**Discussion by jbionic2016** [community/community#129427] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Here is the video link to demonstrate the problem
https://www.youtube.com/watch?v=UOU0VjSTol8

I'd appreciate any hints on how to resolve the issue

**Comment by LiteBrite82** [comment, marked as answer]
It looks like you have created the maximum amount of lists

**Reply by jbionic2016** [reply]
Yep, 32. But would GitHub care to put a cap on the allowed amount of lists? 32 ain't many, 256 looks better

**Comment by jbionic2016** [comment]
So what is the point of this solution?
I don't suppose the issue has to be closed as resolved at this stage.

**Reply by LiteBrite82** [reply]
Hi @jbionic2016 

It was an answer to the title of the discussion - _"Create list" is greyed out in GitHub to add new starred lists_.
And there are multiple discussions that address the feature request for increased list limits / removal of list limits. 

Just trying to keep the boards organized and efficient.

**Reply by jbionic2016** [reply]
What is the point of keeping the board "well organized and efficient" if the reported problems ain't solved?

**Reply by LiteBrite82** [reply]
Its far easier for us to track 100 upvotes on a single discussion than 100 upvotes spread out over many discussions. 

Our standardized reply:

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

**Reply by jbionic2016** [reply]
The link that you shared leads to the page which says
"Making suggestions
We encourage you to "open a discussion" if you have suggestions for how we can improve our products"
That link behind "open a discussion"  in turn leads to the page where my topic has been published
https://github.com/orgs/community/discussions

So here I am, making the feature suggestion..

**Reply by LiteBrite82** [reply]
.... Epic Fail... I see what you are saying, it does kind of create a loop... Though do be sure to read the explanations and instructions.

**Reply by LiteBrite82** [reply]
side note: I loved your video demo - soooo helpful

**Reply by jbionic2016** [reply]
All I expect is that GitHub goes slightly beyond the formalities of chit-chatting and demonstrates a constructive approach to solving problems 

The required change is obviously very minor and can be implemented fast

**Reply by LiteBrite82** [reply]
We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/orgs/github/projects/4247/views/1). While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.
Regarding your specific request, it’s important to note that it’s not actively being worked on at the moment. However, this doesn’t mean we won’t consider it in the future; it’s simply not on our development roadmap right now.
We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.
Your contributions are instrumental in making GitHub better for all users. Thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is truly valuable and contributes to the continuous improvement of GitHub.

**Reply by jbionic2016** [reply]
Thanks. I do find that GitHub is great
